src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -224,16 +216,21 @@ function createCampaignToken(_, playerColor, _)
     return
   end
 
+  -- clean up chaos tokens
+  blessCurseApi.removeAll(playerColor)
+  chaosBagApi.releaseAllSealedTokens(playerColor)
+
   local campaignData = {
     box = campaignBox.getGUID(),
     log = campaignLog.getData(),
     bag = chaosBagApi.getChaosBagState(),
     trauma = traumaValues,
     decks = deckImporterApi.getUiState(),
     clueCount = playAreaApi.getInvestigatorCount(),
-    guide = campaignGuide.Book.getPage(),
+    playmat = playAreaApi.getSurface(),
```

Comment:
This should not be named playmat (because it could be confused with the actual playermats)